src/utils/_math-constants.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * The minimum value beneath which underflow to zero occurs.
+ * @type {Number}
+ */
+const MIN_MATH_DELTA = 1e-7;
```

Comment:
Why are these constants needed in a separate module, exactly? Do we require them somewhere else?

Reply:
Exactly, these constants are used in `round-number` and `sum-prop` functions.